cost/google/cud_recommendations/CHANGELOG.md

```diff
@@ -3,3 +3,24 @@
 ## v2.0
 
 - Initial Release
+
+## v3.0
```

Comment:
think this should be v2.1 as opposed to v3.0

Reply:
it has a bunch of label changes, for the other cloud cost policies we just went ahead with a version upgrade.